src/pages/study-room/hooks/useStudyRoomQuery.ts

```diff
@@ -1,13 +1,25 @@
-import { getStudyRoomInfo } from '@/apis/study-room/get-study-room-info';
 import { useQuery } from '@tanstack/react-query';
+import { StudyRoomInfo } from '@/types/study-room.types';
+import { getStudyRoomInfo } from '@/apis/study-room/get-study-room-info';
 
 interface UseStudyRoomQuery {
   studyId: number;
 }
+
+const initStudyRoom: StudyRoomInfo = {
```

Comment:
init은 동사처럼 쓰이기도 해서 initial로 표현해주면 어떨까싶어요!
```suggestion
const initialStudyRoom: StudyRoomInfo = {
```